README.md

```diff
@@ -27,10 +27,23 @@ Response
 - response model: Comment - The newly created comment object.
 
 ### GET /comments
-Retrieves all comments from the database.
+
+Retrieves comments from the database. It supports filtering by various parameters.
+
+Request Query Parameters
+
+project_name: The name of the project to filter by.  
+feature_url: The URL of the feature to filter by.  
+user_id: The ID of the user to filter by.  
+timestampbegin: The minimum timestamp of the comments to filter by (in ISO 8601 format).  
+timestampend: The maximum timestamp of the comments to filter by (in ISO 8601 format).  
+content_search: A search query to filter comments by. It searches in both the feature URL and comment text fields.  
```

Comment:
No need to search in feature URL, just comment text
Say explicitly you can use a regex